put_cache.py

```diff
@@ -0,0 +1,46 @@
+import time
+import commands
+import os
+
+txn_num = 500
+
+# 0. send milestone
+ret = os.system("java -jar ../../private-iota-testnet/target/iota-testnet-tools-0.1-SNAPSHOT-jar-with-dependencies.jar Coordinator localhost 14700")
```

Comment:
1, should not dependent on outside env;
2, we should fork a private-iota-testnet of our own, or else if they change their code or format, it won't be compatible with ours.